docs/summon-provider.md

```diff
@@ -7,13 +7,13 @@ It is used to execute a process and inject secrets from a separate store. Using
 
 ## Summon Provider
 
-gopass can be used as [summon provider](https://cyberark.github.io/summon/#providers) out of the box, since it fulfills the summon provider contract.
+The gopass repository contains the [cmd/gopass-summon-provider](https://github.com/gopasspw/gopass/tree/master/cmd/gopass-summon-provider) tool, that can be used as [summon provider](https://cyberark.github.io/summon/#providers).
```

Comment:
Please use the [github relative links](https://github.blog/2013-01-31-relative-links-in-markup-files/) here.